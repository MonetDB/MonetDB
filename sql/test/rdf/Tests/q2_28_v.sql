WITH A(subj, obj) as (SELECT * FROM prop17582582_pso where obj = 14660332)
SELECT * from (
		( SELECT 'prop14657017_pso', count(*)
			FROM A, prop14657017_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657018_pso', count(*)
			FROM A, prop14657018_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657019_pso', count(*)
			FROM A, prop14657019_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657202_pso', count(*)
			FROM A, prop14657202_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657206_pso', count(*)
			FROM A, prop14657206_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657207_pso', count(*)
			FROM A, prop14657207_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657210_pso', count(*)
			FROM A, prop14657210_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657214_pso', count(*)
			FROM A, prop14657214_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657215_pso', count(*)
			FROM A, prop14657215_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657217_pso', count(*)
			FROM A, prop14657217_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657220_pso', count(*)
			FROM A, prop14657220_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657222_pso', count(*)
			FROM A, prop14657222_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657234_pso', count(*)
			FROM A, prop14657234_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657240_pso', count(*)
			FROM A, prop14657240_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657244_pso', count(*)
			FROM A, prop14657244_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14657248_pso', count(*)
			FROM A, prop14657248_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659533_pso', count(*)
			FROM A, prop14659533_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659603_pso', count(*)
			FROM A, prop14659603_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659613_pso', count(*)
			FROM A, prop14659613_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659708_pso', count(*)
			FROM A, prop14659708_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659801_pso', count(*)
			FROM A, prop14659801_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659802_pso', count(*)
			FROM A, prop14659802_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14659939_pso', count(*)
			FROM A, prop14659939_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14660087_pso', count(*)
			FROM A, prop14660087_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14660290_pso', count(*)
			FROM A, prop14660290_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14660306_pso', count(*)
			FROM A, prop14660306_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop14660329_pso', count(*)
			FROM A, prop14660329_pso as B
			WHERE A.subj = B.subj 
		)
	 UNION ALL
		( SELECT 'prop17582582_pso', count(*)
			FROM A, prop17582582_pso as B
			WHERE A.subj = B.subj 
		)
 ) as trip;
