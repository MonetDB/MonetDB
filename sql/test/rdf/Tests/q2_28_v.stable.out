stdout of test 'q2_28_v` in directory 'sql/test/rdf` itself:


# 10:46:19 >  
# 10:46:19 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=36263" "--set" "mapi_usock=/var/tmp/mtest-3472/.s.monetdb.36263" "--set" "monet_prompt=" "--forcemito" "--dbpath=/ufs/sjoerd/@Monet-stable/var/MonetDB/mTests_sql_test_rdf"
# 10:46:19 >  

# MonetDB 5 server v11.27.12 (hg id: 9a05add39d7b)
# This is an unreleased version
# Serving database 'mTests_sql_test_rdf', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 62.695 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2018 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://methuselah.da.cwi.nl:36263/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-3472/.s.monetdb.36263
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded


# 10:46:19 >  
# 10:46:19 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-3472" "--port=36263"
# 10:46:19 >  

#WITH A(subj, obj) as (SELECT * FROM prop17582582_pso where obj = 14660332)
#SELECT * from (
#		( SELECT 'prop14657017_pso', count(*)
#			FROM A, prop14657017_pso as B
#			WHERE A.subj = B.subj 
#		)
#	 UNION ALL
#		( SELECT 'prop14657018_pso', count(*)
#			FROM A, prop14657018_pso as B
#			WHERE A.subj = B.subj 
#		)
#	 UNION ALL
#		( SELECT 'prop14657019_pso', count(*)
#			FROM A, prop14657019_pso as B
#			WHERE A.subj = B.subj 
#		)
#	 UNION ALL
#		( SELECT 'prop14657202_pso', count(*)
#			FROM A, prop14657202_pso as B
#			WHERE A.subj = B.subj 
% .trip,	.trip # table_name
% %7,	%3 # name
% char,	bigint # type
% 16,	1 # length
[ "prop14657017_pso",	0	]
[ "prop14657018_pso",	0	]
[ "prop14657019_pso",	0	]
[ "prop14657202_pso",	0	]
[ "prop14657206_pso",	0	]
[ "prop14657207_pso",	0	]
[ "prop14657210_pso",	0	]
[ "prop14657214_pso",	0	]
[ "prop14657215_pso",	0	]
[ "prop14657217_pso",	0	]
[ "prop14657220_pso",	0	]
[ "prop14657222_pso",	0	]
[ "prop14657234_pso",	0	]
[ "prop14657240_pso",	0	]
[ "prop14657244_pso",	0	]
[ "prop14657248_pso",	0	]
[ "prop14659533_pso",	0	]
[ "prop14659603_pso",	0	]
[ "prop14659613_pso",	0	]
[ "prop14659708_pso",	0	]
[ "prop14659801_pso",	0	]
[ "prop14659802_pso",	0	]
[ "prop14659939_pso",	0	]
[ "prop14660087_pso",	0	]
[ "prop14660290_pso",	0	]
[ "prop14660306_pso",	0	]
[ "prop14660329_pso",	0	]
[ "prop17582582_pso",	0	]

# 10:46:19 >  
# 10:46:19 >  "Done."
# 10:46:19 >  

